### DISCUSSION TITLE
Getting Warning 'This CKEditor 4.22.1 version is not secure. Consider upgrading to the latest one, 4.24.0-lts

### DISCUSSION THREAD

**Discussion by MinalRaorane** [serenity-is/Serenity#7197] [Help]
I am getting following warning while I open form and I have HtmlNoteEditor for saving comment
This CKEditor 4.22.1 version is not secure. Consider upgrading to the latest one, 4.24.0-lts: https://ckeditor.com/ckeditor-4-support

![image](https://github.com/serenity-is/Serenity/assets/144375037/64a3b181-2a9a-4488-87d9-645101881c2e)

**Comment by MinalRaorane** [comment]
I found one link as follows giving solution to make 
We have to add versionCheck: false to the ckeditor configuration - see https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-versionCheck

But I don't know how to acheive this serenity.

**Comment by volkanceylan** [comment]
In the latest version of Serenity its off by default for HtmlContentEditor, update Serenity.

**Reply by ra-bedo11** [reply]
How to Update serenity ?
